src/hotspot/share/oops/method.cpp

```diff
@@ -818,6 +818,20 @@ bool Method::can_be_statically_bound(InstanceKlass* context) const {
   return (method_holder() == context) && can_be_statically_bound();
 }
 
+/**
+ *  Returns false if this is one of specially treated methods for
+ *  which we have to provide stack trace in throw in compiled code.
+ *  Returns true otherwise.
+ */
+bool Method::can_omit_stack_trace() {
+  if (method_holder()->class_loader_data()->is_boot_class_loader_data()) {
```

Comment:
Do you actually need to check this?

Reply:
@dholmes-ora, thank you for looking on it.
I discussed it with Mandy and agreed that we need to narrow down this workaround as much as possible. That is why it is done only for system class loaded by null loader.

Reply:
David has a good observation.   There will be no split package for modules.  So sun.invoke.util classes will only be loaded from java.base.  The boot loader is not needed.

Reply:
Okay, I will remove it.